scania-digital-design-system/sdds
# Migrate chips component

**Describe pull-request**  
Migration of chips component (Native)

**Solving issue**  
Fixes: AB#2394

**How to test**  
_Add description how to test if possible_
1. Go to the storybook link below
2. Check in components -> chips
3. Check that the design aligns with figma.

**Screenshots**  
-

**Additional context**  
-
